### DISCUSSION TITLE
Need help with 'Error occurred running finally' at end of run + disabling color output.

### DISCUSSION THREAD

**Discussion by cinnion** [nektos/act#2511] [Q&A]
I am trying to use nektos/act which I cloned and built on Rocky 9 to run the workflow for [cobbler/cobbler](https://github.com/cobbler/cobbler), and at the end of the run (just running the command `act`), where I would expect to see a summary of failed jobs, I instead see:

```
Error: Error occurred running finally: Error occurred running finally: Error occurred running finally: exitcode '1': failure (original error: <nil>) (original error: <nil>) (original error: <nil>)
```

Is this something wrong with act, the repo and its workflow, or my build and the environment?

Also, I have tried setting `export TERM=dumb` before running act, and I still get the colors, the escape sequences for which litter the captured log. Is there any way to turn those off?

**Comment by ChristopherHX** [comment]
`CLICOLOR_FORCE=0`
https://github.com/nektos/act/blob/0de940b948e0397c2db8f98b82dd20d88a542325/pkg/runner/logger.go#L246

**Reply by cinnion** [reply]
OK... first time I have seen this as a way to turn off colors, since in 30 years, the `TERM=dumb` trick has worked for everything from programs written with curses to more modern solutions for CLI programs. But thank you for the solution. If only it were also a flag which could be added to the RC file... but I can certainly add it to my profile.

**Comment by ChristopherHX** [comment]
> `Error: Error occurred running finally: Error occurred running finally: Error occurred running finally: exitcode '1': failure (original error: <nil>) (original error: <nil>) (original error: <nil>)`

The error must be above this line, someone would need to work on improving logs for getting any better. Currently is certainly no one working on that.

**Reply by cinnion** [reply]
I will be running the jobs manually one by one, and will see if it is isolated to a single workflow task, or all of them.
